src/DayTime.cpp

```diff
@@ -153,14 +153,14 @@ void DayTime::addHours(float deltaHours)
 
 void DayTime::checkHours()
 {
-    while (totalSeconds >= secondsPerDay)
+    while (totalSeconds >= SIDEREAL_SECONDS_PER_DAY)
```

Comment:
This must be reverted. This needs to be actual seconds, not sidereal.

Reply:
done